src/jest.mli

```diff
@@ -14,11 +14,11 @@ module Runner (A : Asserter) : sig
   external describe : string -> (unit -> unit) -> unit = "" [@@bs.val]
 
   external beforeAll : (unit -> unit) -> unit = "" [@@bs.val]
-  val beforeAllAsync : ((unit -> unit) -> unit) -> unit
+  val beforeAllAsync : ?timeout:float -> ((unit -> unit) -> unit) -> unit
```

Comment:
It makes more sense to have `timeout` typed as `int` I think. It's in milliseconds, and I doubt we need fractions of those, or to wait long enough to overflow a 32-bit integer :)

Reply:
Fixed, added `timeout` typed as `int`.